### DISCUSSION TITLE
Include x86-64 build for macOS in GH Workflows

### DISCUSSION THREAD

**Discussion by gregerolsson** [bjornbytes/lovr#618] [Q&A] [answered]
Would it be possible to add an automated build for macOS x86-64 in the GitHub Workflow file? Currently the macOS release is for ARM64/Apple Silicon only and can not be run on an x86 machine:

**0.16.0**

```sh
LÖVR.app/Contents/MacOS
❯ file lovr
lovr: Mach-O 64-bit executable arm64
```

The automated 0.15.0 release is built for x86:

**0.15.0**

```sh
LÖVR.app/Contents/MacOS
❯ file lovr
lovr: Mach-O 64-bit executable x86_64
```

**Comment by bjornbytes** [comment, marked as answer]
Thanks for bringing this up, I totally spaced on the x86/arm split.  Here's what I think the current situation is:

- I built the v0.16.0 macOS release on an M1 CPU.  This was probably a mistake, as it doesn't work on a lot of machines.
- GitHub actions was added afterwards and is currently producing x86_64 binaries.  GitHub actions doesn't support M1 yet.

Also, currently the macOS release [requires MoltenVK](#610) to run properly.  LÖVR should probably package that library in the app.

I'm looking into these as I get access to hardware, which currently isn't very often.
